<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAnnotationCommonPropertiesWidgetBase</class>
 <widget class="QWidget" name="QgsAnnotationCommonPropertiesWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Rendering</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Z-index</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsSpinBox" name="mSpinZIndex">
        <property name="minimum">
         <number>-9999999</number>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="mCalloutCheckBox">
       <property name="text">
        <string>Show callout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mCalloutPropertiesButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="mReferenceScaleGroup">
     <property name="title">
      <string>Reference Scale</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QgsScaleWidget" name="mReferenceScaleWidget" native="true">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Minimum scale, i.e. most &quot;zoomed out&quot;.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsScaleWidget</class>
   <extends>QWidget</extends>
   <header>qgsscalewidget.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
